src/thunderbots/software/ai/navigator/trespass.cpp

```diff
@@ -1,8 +1,59 @@
 #include "ai/navigator/trespass.h"
 
+#include <exception>
+
+#include "geom/circle.h"
+#include "geom/point.h"
+#include "util/math_functions.h"
+
 namespace Navigator
 {
     namespace Trespass
     {
-    }
+        // Don't know where to put this???
```

Comment:
I think the proper namespace would probably be `AI::Navigator`. I would add the `AI` at the start because we're in the AI stuff, but I think `Trespass` is probably a little more granular then we want to be.